README.md

```diff
@@ -1,69 +1,77 @@
-# Dependency Injection Checker
+# Guice Checker
 
-A common problem when programming is TODO.
-This results in a run-time exception.
-
-The Dependency Injection Checker guarantees, at compile time, that your code will
-not suffer that run-time exception.
+One common challenge is dealing with dependencies and ensuring they are properly defined. Omitting or misconfiguring dependencies can lead to runtime errors, impacting the reliability and stability of code.
 
+The Guice Checker is a static analysis tool designed to address this issue by providing compile-time guarantees that your Guice dependency mappings are accurately defined.
 
 ## How to run the checker
 
-First, publish the checker to your local Maven repository by running
-`./gradlew publishToMavenLocal` in this repository.
+First, build the checker by running `./gradlew clean build` and then publish the checker to your local Maven repository by running `./gradlew publishToMavenLocal` in this repository.
 
 Then, if you use Gradle, add the following to the `build.gradle` file in
 the project you wish to type-check (using Maven is similar):
 
 ```
-repositories {
-    mavenLocal()
-    mavenCentral()
+plugins {
+    id 'org.checkerframework' version '0.6.28'
+    ...
+}
+
+...
+
+apply plugin: 'org.checkerframework'
+checkerFramework {
+  checkers = [
+    'org.checkerframework.checker.dependencyinjection.DependencyInjectionChecker',
```

Comment:
Given that you are renaming this to the "Guice Checker" in the README, would it make sense to also rename the checker itself to `org.checkerframework.checker.dependencyinjection.GuiceChecker` or something similar? I think so.

Reply:
I think either would be fine.

Reply:
So what you're envisioning are multiple checkers that exist in this `dependencyinjection` directory, with `GuiceChecker` being the first one in there?

This would also mean that any file previously named `DependencyInjection*` would change to `Guice*`, such as `GuiceChecker.java`, right?

Reply:
> This would also mean that any file previously named DependencyInjection* would change to Guice*, such as GuiceChecker.java, right?

Yes.

Reply:
Okay sounds good. I'll create a separate branch for those changes since they'll touch many files and it's a pretty important change.